samples/node-token-validation-samples/basic-sample/README.md

```diff
@@ -0,0 +1,39 @@
+# Sample for Token Validation prototype
+
+This is a sample that can run `node-token-validation`. Note that this is not a finished prototype or a finished sample, and is used for experimental purposes only.
+
+## Instructions to run prototype
+
+### In the Azure Portal
+
+1. Set up or use an existing SPA application.
+
+1. Create a custom scope. Make note of the tenant id, client id, and custom scope.
+
+### In a msal-browser sample (e.g. VanillaJSTestApp2.0)
+
+1. Replace client id, authority, and token request scopes with information from above app registration. Note that token request scopes should only have the custom scope, and no MS Graph scopes, otherwise you will get a token that cannot be validated.
+
+1. Run the sample, sign in, and click get profile.
+
+1. In the developers console, get the secret from the access token that has the custom scope (not Graph scopes).
+
+### In the src/experimental/node-token-validation POC library
```

Comment:
Some of this doesn't match the rest of the PR. Seems like this README was written for a previous iteration?

Reply:
Thanks for the catch. Updated this!